source/uri-options/tests/srv-options.yml

```diff
@@ -24,39 +24,39 @@ tests:
           srvMaxHosts: 2
     - description: "SRV URI with negative integer for srvMaxHosts"
       uri: "mongodb+srv://test1.test.build.10gen.cc/?srvMaxHosts=-1"
-      valid: false
+      valid: true
       warning: true
       hosts: ~
       auth: ~
       options: {}
     - description: "SRV URI with invalid type for srvMaxHosts"
       uri: "mongodb+srv://test1.test.build.10gen.cc/?srvMaxHosts=foo"
-      valid: false
```

Comment:
I was expecting:

```
valid: false
warning: false
```

Reply:
Same as above. "foo" is an invalid value for `srvMaxHosts`, since it is documented as expecting a non-negative integer.